daml-assistant/daml-helper/src/DamlHelper.hs

```diff
@@ -109,6 +110,21 @@ runNew targetFolder templateName = do
         exitFailure
     copyDirectory templateFolder targetFolder
 
+    -- update daml.yaml
+    let configPath = targetFolder </> projectConfigName
+        configTemplatePath = configPath <.> "template"
+
+    whenM (doesFileExist configTemplatePath) $ do
+        configTemplate <- readFileUTF8 configTemplatePath
+        sdkVersion <- getSdkVersion
+        let projectName = takeFileName (dropTrailingPathSeparator targetFolder)
+            config = replace "__SDK_VERSION__"  sdkVersion
```

Comment:
Given that we use __VERSION__ everywhere else it might be a good idea to use it here as well. In general, I prefer `__SDK_VERSION__` but having two different names seems more confusing than using `__VERSION__` consistently until we change everything.

Reply:
Ok! Thanks!